components/Footer/Footer.tsx

```diff
@@ -0,0 +1,86 @@
+import clsx from "clsx";
+import Image from "next/image";
+import Link from "next/link";
+import { HTMLAttributes } from "react";
+
+import { usePaths } from "@/lib/paths";
+import { MenuItemFragment, useFooterMenuQuery } from "@/saleor/api";
+
+import { Box } from "../Box";
+import { useRegions } from "../RegionsProvider";
+import styles from "./Footer.module.css";
+
+export type FooterProps = HTMLAttributes<HTMLElement>;
+
+export function Footer({ className, ...rest }: FooterProps) {
+  const paths = usePaths();
+  const { query } = useRegions();
+
+  const { data, error } = useFooterMenuQuery({ variables: { ...query } });
+
+  if (error) {
+    console.error("Footer component error", error.message);
+    return null;
+  }
+
+  const menu = data?.menu?.items || [];
+
+  const gethLinkPath = (item: MenuItemFragment) => {
```

Comment:
WDYT about moving this function to the `lib/menus.ts`, it seems like the main menu would use the same code?

Reply:
Done.